src/web/tests.py

```diff
@@ -325,6 +325,160 @@ def test_check_listing_massage_views(self):
+                                        "duration": 3600,
+                                        "created": "2023-03-31 15:15:50",
+                                    }
+                                ],
+                                "status": massage2.status,
+                                "serviceId": massage2.service.external_id,
+                                "providerId": massage2.therapist.userprofile.external_id,
+                                "bookingStart": massage2.start.strftime(
+                                    "%Y-%m-%d %H:%M:%S"
+                                ),
+                                "bookingEnd": massage2.end.strftime(
+                                    "%Y-%m-%d %H:%M:%S"
+                                ),
+                            },
+                        ],
+                    },
+                    "2023-04-07": {
+                        "date": "2023-04-07",
+                        "appointments": [
+                            {
+                                "id": 579,
+                                "bookings": [
+                                    {
+                                        "id": 345,
+                                        "customerId": 333,
+                                        "customer": {
+                                            "id": customer3.external_id,
+                                            "firstName": customer3.name,
+                                            "lastName": customer3.surname,
+                                            "email": customer3.email,
+                                            "phone": customer3.phone,
+                                        },
+                                        "status": massage3.status,
+                                        "price": massage3.service.price,
+                                        "appointmentId": massage3.external_id,
+                                        "persons": 1,
+                                        "duration": 3600,
+                                        "created": "2023-03-31 15:15:50",
+                                    }
+                                ],
+                                "status": massage3.status,
+                                "serviceId": massage3.service.external_id,
+                                "providerId": massage3.therapist.userprofile.external_id,
+                                "bookingStart": massage3.start.strftime(
+                                    "%Y-%m-%d %H:%M:%S"
+                                ),
+                                "bookingEnd": massage3.end.strftime(
+                                    "%Y-%m-%d %H:%M:%S"
+                                ),
+                            },
+                        ],
+                    },
+                },
+            },
+        }
+
+        customer_import(data)
+        massage_import(data)
```

Comment:
Why does this test need importer at all, why not just use Customer Factory?

Reply:
needs discussion...